src/osx/carbon/utilscocoa.mm

```diff
@@ -613,6 +613,27 @@ void  wxMacCocoaShowCursor()
 {
     [NSCursor unhide];
 }
+
+WX_NSCursor wxMacCocoaCreateCursorFromResource(const wxString &cursor_file, wxBitmapType flags)
+{
+    NSString *path;
+    NSBundle *bundle = [NSBundle mainBundle];
+    if( flags == wxBITMAP_TYPE_MACCURSOR_RESOURCE )
```

Comment:
I wouldn't say a MACCURSOR is typically a .CUR file ... this rang belongs to .PNG IMHO

Reply:
@csomor,
I can make it to work with PNG.
I was not sure if the "CUR" file can be easily converted to "PNG".

I will also see if I can address other comments.